Prague 2022 blog post - Get ready (#1796)
writethedocs/www: docs/conf/prague/2022/news/get-ready.rst

:template: {{year}}/generic.html

.. post:: Sept 1, 2022
   :tags: {{year}}, {{shortcode}}-{{year}}, tickets

Get Ready for the conference!
=============================

Hey everyone!

> We're just over a week away from the start of the conference, and we're getting excited to see you soon! We're looking forward to gather for 3 days of content, collaboration, and conversation.

Author: ```suggestion
We're just under a week away from the start of the conference, and we're getting excited to see you soon! We're looking forward to gather for 3 days of content, collaboration, and conversation.
```

We have a few reminders before we welcome you all next Sunday:

> * Ticket sales close next week, so grab yours now!

Author: ```suggestion
* Ticket sales close this week, so grab yours now!
```

* We have a great Welcome Wagon, which has a guide, as well as introductory tours of our online platform
* There are lots of ways to get involved at the conference on the day-of, which we list out below so you can prep a bit for them.

> Ticket sales close on **September 8th**

Author: ```suggestion
Ticket sales close on **Thursday September 8th**
```

> Ticket sales close on **Thursday September 8th**
> ---------------------------------------

Author: ```suggestion
Ticket sales close on **Thursday September 8th**
------------------------------------------------
```

> To help us manage capacity and make sure that our conference environment is cozy and friendly, we are going to close ticket sales on **September 8th at 23:59 CEST**.

Author: ```suggestion
To help us manage capacity and make sure that our conference environment is cozy and friendly, we are going to close ticket sales on **Thursday September 8th at 23:59 CEST**.
```

> To help us manage capacity and make sure that our conference environment is cozy and friendly, we are going to close ticket sales on **September 8th at 11:59PM Central European Time**.

Reviewer: ```suggestion
To help us manage capacity and make sure that our conference environment is cozy and friendly, we are going to close ticket sales on **September 8th at 23:59 CEST**.
```
We're not in America anymore with that weird clock ;)
Also, CET is technically UTC+1, we're still in CEST.

If you haven't purchased your ticket yet we invite you to `get your tickets soon <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_.

> For those of you who already purchased your tickets, you will get an email with our online conference platform (Hopin) registration link on September 5th. We will also send a reminder on September 7th, in case you missed the first email.

Reviewer: ```suggestion
For those of you who already purchased your tickets, you will get an email with our online conference platform (Hopin) registration link on September 5th. We will also send a reminder on September 9th, in case you missed the first email.
```
Reminder is on Friday, so it also catches all last-minute registrations.

This will give you the opportunity to set up your Hopin account and troubleshoot any access issues that you might have.

If you have a ticket and don't get an email from us by September 7th, please email us and we'll help you out!

Welcome Wagon guide and platform tour
-------------------------------------

Each year we create a guide to help our attendees get the most out of the conference.
Following the success of last year's conferences and based on your feedback, we expanded `our Welcome Wagon guide <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/welcome-wagon/>`_ to help you get the most out of our event this year.

> Another valuable thing that we're doing this year is the Welcome Wagon team will hold **live orientation sessions and platform tours** for our virtual platform, Hopin. **There will be 2 sessions on Sunday, and 1 on Monday morning.**  This will give you a chance to walk through the virtual event and ask all your questions ahead of the conference.

Reviewer: ```suggestion
Another valuable thing that we're doing this year is the Welcome Wagon team will hold **live orientation sessions and platform tours** for our virtual platform, Hopin. **There will be two sessions on Sunday, and one on Monday morning.**  This will give you a chance to walk through the virtual event and ask all your questions ahead of the conference.
```

Check out the `Schedule page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/schedule/>`_ for exact time and locations.

All our events and activities
-----------------------------

There are a large number of events at the conference beyond the main track of speakers.
We hope that you'll get involved with some of our other events,
and get to know more folks that are hanging out at the conference!

* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/lightning-talks/>`__ each day after lunch. Registration is handled on the day of the conference. These are 5-minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.

> * **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/unconference/>`_ all day Monday and on Tuesday afternoon. These are also planned day-of, and allow attendees to schedule a session to talk about any topic they wish. It's a great way to find others who share your specific interests.

Reviewer: ```suggestion
* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/unconference/>`_ all day Monday and on Tuesday afternoon. These are also planned day-of, and allow attendees to schedule a session to talk about any topic they wish. It's a great way to find others who share your specific interests. As a session lead you guide the session, but you definitely do not need to have a full plan.
```
Or something along those lines. Might lower the unconference barrier without lowering it too much.

* **Conference Help Desk**. If you have any questions about navigating the virtual event, we'll be running the Write the Docs Helpdesk booth in the **Expo** area, where you can chat with one of the organizers and the Welcome Wagon team.

The `Schedule page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/schedule/>`_ has the latest and most complete information about all our events. This is where we will also update any changes or cancellations if they should occur.

Thanks so much to all our sponsors
----------------------------------

We are so grateful to have our sponsors help in bringing these events to life every year.
You'll be able to chat with most of our sponsors at the Job Fair, and a few will have booths for conversations the entire conference.

Thanks so much to the following companies for supporting the Portland conference this year:

.. datatemplate::
   :source: /_data/{{shortcode}}-{{year}}-config.yaml
   :template: {{year}}/sponsors-simplelist.rst

> 

Reviewer 2: Sorry folks, haven't had a chance to look at this, #1798 contains a speaker name change, so would be better to merge that before this one.
